src/Core/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 ### Fixed
 
+- remove code from SignerV4 dedicated to S3 requests.
```

Comment:
Maybe we should explain the bug better. Ie “fixed invalid signatures”. And explain why it is a bug fix.

Reply:
```suggestion
- Fixed invalid chunking of request body. When a large request body was sent, we must not split it up to multiple parts. This causes issues on most clients. It is the preferred way for the S3Client. This version removed the invalid code from SignerV4 to make sure requests are not chunked. 
```

Reply:
I see that you updated the comment. Maybe use a combination of our suggestions?

Reply:
I've shorten a little bit your sentence (not sure changelog is the place for detailed explanation)